src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/DocumentTransformer.kt

```diff
@@ -63,20 +25,4 @@ class DocumentTransformer {
           description = it.description,
         )
       }
-
-  /**
-   * If a document doesn't have an ID we have no way to retrieve it for download, so it's filtered out
-   *
-   * If a document doesn't have a name, we have no way to determine what the file should be called on download,
-   * or what file extension (e.g. pdf) should be used. This rarely occurs.
-   */
-  private fun documentsWithIdsAndNames(documents: List<CommunityApiDocument>): List<CommunityApiDocument> {
```

Comment:
@gregkhawkins i can't recall if we discussed the 'loss' of this logic when moving away from community api. Shall we have a quick discussion when you're back.

@vbala-moj this doesn't block merging